schema/assets/image.json

```diff
@@ -44,7 +44,8 @@
                     "if": {
                         "properties": {
                             "e": {"const": 1}
-                        }
+                        },
+                        "required": ["e"]
```

Comment:
how do schema validators handle missing `e` without `required` here?

Reply:
`e` is considered optional so it will pass the `if` schema and evaluate the `then` schema. This is somewhat unintuitive but is how JSON schema spec is designed. I know I've overlooked this a few times.